### PR TITLE
chore: remove git repository orchestration

### PR DESCRIPTION
**Pull request by ferrarimarco**
Don't use Terraform to orchestrate operations on the Config Sync Git repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,18 @@ Users and teams managing tenant apps should not have permissions to change clust
 ## Deploy the blueprint
 
 1. Open [Cloud Shell](https://cloud.google.com/shell)
-1. Clone this repository
+
+1. Initialize the local repository where the environment configuration will be
+   stored:
+
+   ```sh
+   ACM_REPOSITORY_PATH= # Path on the host running Terraform to store environment configuration
```
Comment (laurentgrangeau) [inline README.md:118]: As we get rind of reference to Anthos, maybe rename these variables to CONFIG_SYNC ?
https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs

File: README.md [hunk 2]
```diff
@@ -129,8 +140,9 @@ Users and teams managing tenant apps should not have permissions to change clust
 
    ```hcl
    project_id                  = # Google Cloud project ID where to provision resources with the blueprint.
-   acm_repository_path         = # Path on the host running Terraform to store environment configuration
-   acm_repository_url          = # URL of the repository to store environment configuration
+   acm_branch                  = # Use the same value that you used for ${ACM_BRANCH}
```
Comment (laurentgrangeau) [inline README.md:143]: See previous comment

**Review by laurentgrangeau — COMMENTED** [review]
Minor things to review

File: README.md (hunk 1 above)
Comment (ferrarimarco) [reply README.md:118]: Makes sense, although we can track it in a separate PR because there are many references `acm_...` variables.

**Review by laurentgrangeau — APPROVED** [review]
LGTM

**Pull request merged**